src/main/java/org/graylog/aws/inputs/codecs/CloudWatchFlowLogCodec.java

```diff
@@ -51,6 +51,8 @@ public Message decodeLogData(@Nonnull final CloudWatchLogEvent logEvent) {
                     flowLogMessage.getTimestamp()
             );
             result.addFields(buildFields(flowLogMessage));
+            result.addField("aws_log_group", logGroup);
```

Comment:
Please extract the key names as constants into the existing `AWS` class and use them in `CloudWatchFlowLogCodec` and in `CloudWatchRawLogCodec`.

Reply:
I've extracted these fields to constants.